crates/ethjson/src/state/test.rs

```diff
@@ -110,6 +123,16 @@ pub struct PostStateIndexes {
     pub value: u64,
 }
 
+/// Access list deserialization.
+#[derive(Debug, PartialEq, Deserialize, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct AccessList {
```

Comment:
There is `AccessListItem` from `crates/ethjson/src/blockchain/transaction.rs` that we can reuse here

Reply:
Fixed